### DISCUSSION TITLE
Deeplink for many extensions configuration

### DISCUSSION THREAD

**Discussion by Hastes** [WalletConnect/walletconnect-monorepo#4200] [Developer Support] [answered]
Where should be placed deeplink in dashboard settings for web extension? 

I see desktop linking and placeholder be like myapp://link , but i want to open extension in different browsers. 
For example, i have global link _chrome-extension://nmladckinifchdidkoaagbcpnieocoah/home.html#wc_ , but also i have firefox link, what i need to add and where?

**Comment by glitch-txs** [comment]
we don't support extension wallets this way, only web applications

**Reply by Hastes** [reply]
We wanna add browser tab, like in metamask 

<img width="365" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/19662992/368e656d-0a7f-48e5-89db-67e5f13ed671">

But we don't have any tabs
<img width="388" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/19662992/f5bda0e5-ea4f-4d6c-8761-f7047e4f612f">


What config should i use for firefox and chrome browsers?

**Reply by glitch-txs** [reply]
because MetaMask has two wallets - extension and mobile

**Reply by glitch-txs** [reply, marked as answer]
This will depend on your cloud configuration, you have to flag your wallet as `injected`

**Reply by Hastes** [reply]
thanks, it helps
